src/aleph/vm/orchestrator/cli.py

```diff
@@ -307,6 +303,12 @@ def main():
         FAKE_DATA_PROGRAM=args.fake_data_program,
         DEBUG_ASYNCIO=args.debug_asyncio,
         FAKE_INSTANCE_BASE=args.fake_instance_base,
+        LOG_LEVEL=logging.getLevelName(args.loglevel),
+    )
+
+    logging.basicConfig(
+        level=settings.loglevel,
```

Comment:
Typo ?
```suggestion
        level=settings.LOG_LEVEL,
```

Reply:
You tell me. it's your code I have just been doing rebases